app/main.py

```diff
@@ -1,7 +1,24 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+from requests import Response
+
+
+def get_weather(api_key: str, city: str = "Paris") -> Response:
```

Comment:
Consider adding type hints for the return type of the `get_weather` function to improve code readability and maintainability.